### PR TITLE
Update deps and fix the annoying warning message “Emphasis must use * as marker”

### PR DESCRIPTION
**Pull request by 343dev**

### CODE REVIEW

**Review by igoradamenko — APPROVED** [review]
LGTM.

@dissimulazione, do you mind to check?

File: MIGRATION.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Migration
+
+## 1.2.1 → 2.0.0
+
+No post-upgrade action is required if you are using a linter without a custom
+configuration file.
+
+If you are using a custom configuration file, then you should check the
```
Comment (dissimulazione) [inline MIGRATION.md:8]: I'm not sure, but that line seems weird, maybe it's just Present Simple here, not Continuous?

And I think there's no need in "the" articles here either.

File: MIGRATION.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+[changes](https://github.com/funbox/markdown-lint/commit/e6e3a3fe723f7c7529e7b651639f85ebaa061f85)
+made to the [.markdownlintrc.js](.markdownlintrc.js) in order to avoid
+unexpected behavior.
```
Comment (dissimulazione) [inline MIGRATION.md:11]: An unexpected?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2.0.0 (03.12.2020)
+
+⚠️ Breaking changes.
+
+- Updated `remark` to version
+  [13.0.0](https://github.com/remarkjs/remark/releases/tag/13.0.0).
+- Updated `remark-preset-lint-markdown-style-guide` to version 4.0.0.
+- Updated [.markdownlintrc.js](.markdownlintrc.js) to match the new versions.
```
Comment (dissimulazione) [inline CHANGELOG.md:10]: Maybe there's no need in "the" article here?

**Review by dissimulazione — CHANGES_REQUESTED** [review]

**Review by dissimulazione — APPROVED** [review]

**Comment by igoradamenko** [comment]
Published @funboxteam/markdown-lint@2.0.0.

**Pull request merged**